tf_encrypted/keras/backend.py

```diff
@@ -0,0 +1,59 @@
+"""TFE Keras backend"""
```

Comment:
some of this code seems to come from TF Keras, in which case we should at least mention that in the file header

Reply:
good point